### PR TITLE
Gpu Builder Instance Type prop fix

### PR DESCRIPTION
**Pull request by Howlla**
*Issue #, if available:*

*Description of changes:* Concatenate instanceClass and instanceSize for instanceType prop

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/builders/gpu-builder.ts [hunk 1]
```diff
@@ -158,7 +158,7 @@ function addGpuNodeGroup(options: GpuOptions): clusterproviders.ManagedNodeGroup
     return {
         id: "mng-linux-gpu",
         amiType: NodegroupAmiType.AL2_X86_64_GPU,
```
Comment (elamaran11) [inline lib/builders/gpu-builder.ts:160]: What about `NodegroupAmiType` ? Also please remove the blocksize field as discussed yesterday

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@Howlla Thanks for the fix, couple of minor changes

File: lib/builders/gpu-builder.ts (hunk 1 above)
Comment (Howlla) [reply lib/builders/gpu-builder.ts:160]: I think NodegroupAMIType is better off being fixed. removing it altogether as a GPU Option is the way to go

Comment (elamaran11) [reply lib/builders/gpu-builder.ts:160]: Agree, also remove the blockdevice one

File: lib/builders/gpu-builder.ts [hunk 2]
```diff
@@ -44,10 +40,6 @@ export interface GpuOptions {
      * Optional, Maximum number of nodes to use for the cluster. 
      */
     maxNodeSize?: number,
-    /** 
-     * Optional, Block device size.
-     */
-    blockDeviceSize?: number,
```
Comment (elamaran11) [inline lib/builders/gpu-builder.ts:50]: This should be used and passed as paramter to nodegroup

**Review by elamaran11 — CHANGES_REQUESTED** [review]
BlockSize is needed for nodegroup

**Review by elamaran11 — COMMENTED** [review]
@Howlla Could you please make sure you `tar` the blueprints and test it on patterns. Since there is no e2e for builder classes, we need to test it before we can merge.

**Comment by Howlla** [comment]
Sure

**Comment by Howlla** [comment]
@elamaran11 Tested in the gpu monitoring pattern here https://github.com/aws-samples/cdk-eks-blueprints-patterns/pull/141

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**